mil_common/drivers/mil_pneumatic_actuator/mil_pneumatic_actuator/constants.py

```diff
@@ -33,8 +33,7 @@ class Constants:
     PING_RESPONSE = 0x11
     CHECKSUM_CODE = 0xFF
 
-    @classmethod
-    def create_checksum(cls, byte: int) -> int:
+    def create_checksum(byte: int) -> int:
```

Comment:
Any Python method in a parent class that is not decorated with `@classmethod` or `@staticmethod` should have `self` as the first parameter. [Check out this!](https://docs.python.org/3/tutorial/classes.html#random-remarks)